tests/Rules/data/CorrectFilesystem.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Rules\Data;
+
+use Illuminate\Support\Facades\Storage;
+
+class CorrectFilesystem
+{
+    public function getStorage(): Storage
+    {
+        return Storage::disk('local');
```

Comment:
There's no need to have two separate test files, you can add both lines in this test (rename the file of course, and there's no need for there to be a class) and just assert that the appropriate line throws the error:

The test file can be as simple as:
```php
<?php

declare(strict_types=1);

use Illuminate\Support\Facades\Storage;

function test(): void
{
    Storage::disk('local');
    Storage::disk('this-is-not-defined');
}
```

Reply:
and maybe a variable: `Storage::disk($_GET['disk']);`